src/com/google/javascript/jscomp/CodeGenerator.java

```diff
@@ -1061,7 +1061,7 @@ protected void add(Node n, Context context) {
           }
         }
         add("[");
-        add(first);
+        addExpr(first, NodeUtil.precedence(Token.COMMA) + 1, Context.OTHER);
```

Comment:
Please revert this change, because parentheses aren't needed within the `[]` of the computed property.